### ISSUE TITLE
If no value or placeholder on <select>, the first option should be selected

### ISSUE THREAD

**Issue by hmaesta** [wearebraid/vue-formulate#165]
On native HTML the default value for a `<select>` is the first option.

```html
<select>
   <option>Audi</option>
   <option>Tesla</option>
   <option>Volvo</option>
</select>
```

↳ `Audi` will be selected.

But if using Vue Formulate and the input doesn't have a default value or a placeholder, the `<select>` is blank.

### Proposal
If the `<FormulateInput>` doesn't have a value (from `v-model` or `value` itself) or `placeholder`, it should automatically choose the first option from the object.

Would be the same as:
```vue
<FormulateInput type="select" :options="brands" :value="Object.keys(brands)[0]"/>
```

→ [**Sandbox** comparative & example](https://codesandbox.io/s/vue-formulate-select-default-value-wjznp?file=/src/Formulate.vue)

**Comment by justin-schroeder** [comment]
@hmaesta thanks for the well thought out report. I think I agree with you, provided there is no model defined at the form level or the parent level. As a rule, we never want to mutate a model just to render a field (if possible). I'll mark this as something we need to tackle.

**Comment by justin-schroeder** [comment]
This is now fixed in `v2.4.3`. Thanks @hmaesta.
